### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Astrareach**
french translation

### CODE REVIEW

**Comment by MarcinWieczorek** [comment]
`Fr-Fr.yml` file contains syntax errors. Please commit the fix to branch `patch-2` in order to automatically add it to this Pull Request.
Uhm, could you also rename the file to lowercase letters?

``` plain
Tests in error: 
  testTranslations(co.marcin.novaguilds.yaml.YamlIncompleteTranslationTest)
  testTranslations(co.marcin.novaguilds.yaml.YamlParseTest): Invalid YAML file (./src/main/resources/lang/Fr-Fr.yml)
```

**Comment by MarcinWieczorek** [comment]
The file contains syntax errors.

**Comment by Astrareach** [comment]
I really dont know where it can be, syntax is like special character ?

---

De : Marcin Wieczorek notifications@github.com
Envoyé : samedi 10 septembre 2016 14:10:13
À : MarcinWieczorek/NovaGuilds
Cc : Noiknez; Author
Objet : Re: [MarcinWieczorek/NovaGuilds] French translation (#75)

The file contains syntax errors.

##

**Comment by MarcinWieczorek** [comment]
You need to run maven tests and check if YAML files are correct before you commit anything.
Why add a second apostrophe everywhere?

**Comment by Astrareach** [comment]
Second apostrophe is just an habit because of other plugin langage file to make it work
i dont know why the test in not working ..

**Comment by MarcinWieczorek** [comment]
There are apostrophes in english translation (for sure) and there's no need for two. Have you checked that in-game?

**Comment by Astrareach** [comment]
I made the appostrophe change, if it still doenst work, try with this version http://pastebin.com/BFBMzKgz :) in it there is no special character

**Comment by MarcinWieczorek** [comment]
It's not about characters. There's a missing double quote. Use online yaml parser

File: src/main/resources/lang/Fr-Fr.yml [hunk 1]
```diff
@@ -0,0 +1,726 @@
...
+     points: "&4&l> &7/nga &cg &6<&cguild&6> &cpoints &6<&cpoints&6> &2- &6fixe les points"
+     lives: "&4&l> &7/nga &cg &6<&cguild&6> &clives &6<&clives&6> &2- &6fixe les vies"
+    abandon: "&4&l> &7/nga &cg &6<&cguild&6> &cabandon &2- &6abandonne la guilde"
+    bank:
+     pay: "&4&l> &7/nga &cg &6<&cguild&6> &cpay &6<&camount&6> &2- &6Verse de l'argent à la banque de la guilde"
+     withdraw: "&4&l> &7/nga &cg &6<&cguild&6> &cwithdraw &6<&camount&6> &2- &6Prélève de l'argent dans la banque d'une guilde"
+    purge: "&4&l> &7/nga &cg &6purge &2- &6Supprime toutes les guiles"
+    tp: "&4&l> &7/nga &cg &6<&cguild&6> &ctp &6[&cplayer&6] &2- &6TP au foyer de la guilde"
+    reset:
+     points: "&4&l> &7/nga &cg &6resetpoints &6<&ccondition&6> &6[&cvalue&6] &2- &6Réinitialise les points de la guilde
```
Comment (MarcinWieczorek) [inline src/main/resources/lang/Fr-Fr.yml:153]: missing "

**Comment by MarcinWieczorek** [comment]
Also please add a line in the config like this one: d9f56c1. I want this PR to be fully valid before I merge the changes.

Also this is the parser I've been using for years: http://yaml-online-parser.appspot.com/

**Comment by Astrareach** [comment]
I understand, i will get back to you soon

**Pull request merged**